Sources_v3/StreamChatUI/Utils/NSLayoutConstraint+Extensions.swift

```diff
@@ -4,9 +4,17 @@
 
 import UIKit
 
+extension UILayoutPriority {
+    static let almostRequired: Self = .required - 1
+}
+
 extension NSLayoutConstraint {
     func with(priority: UILayoutPriority) -> NSLayoutConstraint {
         self.priority = priority
         return self
     }
+
+    var almostRequired: NSLayoutConstraint {
+        with(priority: .almostRequired)
+    }
```

Comment:
nit: when introducing helpers like this, please always add a comment explaining what's happening and how/where to use ti.

Reply:
added in 3ab33bb2ab5e73b90518d92f655579839d03eabe